### PR TITLE
logging: fix the broken build

### PR DESCRIPTION
**Pull request by rakyll**

### CODE REVIEW

File: logging/simplelog/simplelog.go [hunk 1]
```diff
@@ -35,13 +35,19 @@ func main() {
...
-		option.WithScopes(logging.AdminScope))
-
+	client, err := logging.NewClient(ctx, projID)
 	if err != nil {
 		log.Fatalf("Failed to create logging client: %v", err)
 	}
+
+	adminClient, err := logadmin.NewClient(ctx, projID,
+		// Admin scope is required to delete logs.
+		option.WithScopes(logging.AdminScope),
```
Comment (jba) [inline logging/simplelog/simplelog.go]: AdminScope is the default for this client.

File: logging/simplelog/simplelog.go [hunk 2]
```diff
@@ -107,29 +113,29 @@ func structuredWrite(client *logging.Client) {
...
 	var entries []*logging.Entry
 	const name = "log-example"
-	iter := client.Entries(ctx,
+	iter := adminClient.Entries(ctx,
 		// Only get entries from the log-example log.
-		logging.Filter(fmt.Sprintf(`logName = "projects/%s/logs/%s"`, projID, name)),
+		logadmin.Filter(fmt.Sprintf(`logName = "projects/%s/logs/%s"`, projID, name)),
 		// Get most recent entries first.
-		logging.OrderBy("timestamp desc"),
+		logadmin.OrderBy("timestamp desc"),
```
Comment (jba) [inline logging/simplelog/simplelog.go]: This is gone, or it should be gone. We have either nothing, or NewestFirst()

File: logging/simplelog/simplelog_test.go [hunk 3]
```diff
@@ -21,9 +22,13 @@ func TestSimplelog(t *testing.T) {
 	tc := testutil.SystemTest(t)
 	ctx := context.Background()
 
-	client, err := logging.NewClient(ctx, tc.ProjectID, option.WithScopes(logging.AdminScope))
+	client, err := logging.NewClient(ctx, tc.ProjectID)
 	if err != nil {
-		t.Fatalf("NewClient: %v", err)
+		t.Fatalf("logging.NewClient: %v", err)
+	}
+	adminClient, err := logadmin.NewClient(ctx, tc.ProjectID, option.WithScopes(logging.AdminScope))
```
Comment (jba) [inline logging/simplelog/simplelog_test.go]: ditto about AdminScope

**Comment by broady** [comment]
Something's importing 1.7 context :/

**Comment by jba** [comment]
Nothing under cloud.google.com/go, AFAICT.

File: logging/simplelog/simplelog.go (hunk 1 above)
Comment (rakyll) [inline logging/simplelog/simplelog.go]: Done

File: logging/simplelog/simplelog.go (hunk 2 above)
Comment (rakyll) [inline logging/simplelog/simplelog.go]: Done

File: logging/simplelog/simplelog_test.go (hunk 3 above)
Comment (rakyll) [inline logging/simplelog/simplelog_test.go]: Done.

**Comment by rakyll** [comment]
All done.

> Something's importing 1.7 context :/

Where? Not in this sample.

**Comment by broady** [comment]
In speech. Fix in this CL?

screen shot 2016-09-28 at 2 04 32 pm

**Comment by rakyll** [comment]
Done.

**Pull request merged**